src/libraries/Native/Unix/System.Globalization.Native/CMakeLists.txt

```diff
@@ -59,22 +59,27 @@ set(NATIVEGLOBALIZATION_SOURCES
 
 include_directories(${UTYPES_H})
 
-add_library(System.Globalization.Native
-    SHARED
-    ${NATIVEGLOBALIZATION_SOURCES}
-    ${VERSION_FILE_PATH}
-)
+if (CORECLR_BUILD)
+    include_directories("../Common")
+endif()
 
-target_link_libraries(System.Globalization.Native
-    dl
-)
+if (GEN_SHARED_LIB)
+    add_library(System.Globalization.Native
+        SHARED
+        ${NATIVEGLOBALIZATION_SOURCES}
+        ${VERSION_FILE_PATH}
+    )
 
-install_library_and_symbols (System.Globalization.Native)
+    target_link_libraries(System.Globalization.Native
+        dl
+    )
+
+    install_library_and_symbols (System.Globalization.Native)
+endif()
 
 add_library(System.Globalization.Native-Static
     STATIC
     ${NATIVEGLOBALIZATION_SOURCES}
-    ${VERSION_FILE_PATH}
```

Comment:
It would be nice to delete it from all static libs for consistency.

Reply:
Also fixed.